lib/migration.js

```diff
@@ -166,6 +175,18 @@ module.exports = function(IBMDB) {
     // ------------------------------------------------------------------------
     sql = sql.concat(self.buildIndexes(model));
 
+    // add back the primary key index
+    // this snippet is required in here rather than the `buildIndexes` function
+    // because DB2 automatically creates an index with a primary key when a
```

Comment:
Yes, I understand that part. I added this comment because **buildIndexes** function cannot be used to alter the primary keys as needed. We need to explicitly put it under **alterTable**.

I will remove the extra comments which creates confusion.